src/typeDefs/AudioStorage.js

```diff
@@ -0,0 +1,40 @@
+import { gql } from "apollo-server-express";
+
+export default gql`
+
+  scalar byteArray

```

Comment:
I think we need to define the custom scalar
because the basic scalar does not include byteArray
@LUXIANZE am I right?

Reply:
I am not sure to use byteArray or JSON will be better

Reply:
basic scalar does not include byteArray nor JSON, need to be custom